packages/ngspice/src-frontend-com_shell.c.patch

```diff
@@ -1,11 +1,20 @@
---- a/src/frontend/com_shell.c
-+++ b/src/frontend/com_shell.c
+--- ngspice-37/src/frontend/com_shell.c	2022-05-17 18:00:52.000000000 +0800
++++ ngspice-37.mod/src/frontend/com_shell.c	2022-06-16 22:56:25.591998228 +0800
 @@ -14,7 +14,7 @@
  #ifdef _WIN32
  #define SHELL "cmd /k"
  #else
 -#define SHELL "/bin/csh"
-+#define SHELL "@TERMUX_PREFIX@/bin/sh"
++#define SHELL "@TERMUX_PREFIX@/bin/csh"
```

Comment:
We don't have csh in our repository. So change this back to sh

Reply:
I did a quick `git reset --soft HEAD^`, made the changes, committed and forced updated the harieamjari branch.